[REQUEST] Multi-User Media Cleaner (#43)
awesome-jellyfin/awesome-jellyfin: README.md

- [wizarr](https://github.com/Wizarrrr/wizarr) - Advanced user invitation and management system.
- [ytdl-sub](https://github.com/jmbannon/ytdl-sub) - Automate downloading and metadata generation with YoutubeDL.
- [xsrv.jellyfin](https://github.com/nodiscc/xsrv/tree/master/roles/jellyfin) - Ansible role to deploy and configure Jellyfin.

> - [Multi-User Media Cleaner](https://github.com/terrelsa13/MUMC) - Query your Jellyfin server for the movies, tv episodes, audio tracks, and audiobooks in your libraries. Then delete media items you no longer want taking up disk space.

Reviewer: I think the description can be a bit shorter, e. g.:

```suggestion
- [Multi-User Media Cleaner](https://github.com/terrelsa13/MUMC) - Query and delete unwanted media content from your Jellyfin server.
```

Author: Changed

> - [Multi-User Media Cleaner](https://github.com/terrelsa13/MUMC) - Query your Jellyfin server for the movies, tv episodes, audio tracks, and audiobooks in your libraries. Then delete media items you no longer want taking up disk space.

Reviewer: Also, please make sure that the order is correct. `Multi-User Media Cleaner` should sit between `jfa-go` and `Playlifin`

Author: Done
